localstack/config.py

```diff
@@ -1016,6 +1014,44 @@ def get_gateway_listen(gateway_listen: str) -> List[HostAndPort]:
 # Whether to return and parse access key ids starting with an "A", like on AWS
 PARITY_AWS_ACCESS_KEY_ID = is_env_true("PARITY_AWS_ACCESS_KEY_ID")
 
+# List of services supported by LocalStack
+SUPPORTED_SERVICES = [
```

Comment:
Wondering if we should enumerate the list of services explicitly here - would be nice if we can somehow integrate this with the service plugin mechanism. (maybe @thrau has some good ideas here..?)

Reply:
I thought about this, but the problem is that this is already used when populating the service specific configs (`<service>_PROVIDER_OVERRIDE` and others) and this list is actually used in controlling the services which should be loaded. That made it a bit too risky for me in the first iteration...
But I am happy for any suggestions and would love to get rid of this list all together!

Reply:
Might this be a way forward? 5766efb103806d463807b136dd9661ffaa350f7f

* the `populate_config_env_var_names` is mostly used for choosing which env vars should be handed to the container. here it makes sense to not use the service plugin manager yet, since this is called in the `config.py`. however, i think it would be OK to be a bit more lenient, and just use a heuristic based on the key syntax. worst case the container sees some variables it doesn't need. i feel this is an acceptable trade off if it means we don't need to maintain a list of supported services.
* The `get_enabled_apis` control path is only used when `EAGER_SERVICE_LOADING` is set to 1, and we've already called `SERVICE_PLUGINS.list_available()` before, so this seems like a no-brainer

Reply:
Great! Thanks a lot for the commit. I just directly cherry-picked it from your branch.